### DISCUSSION TITLE
MAL (Myanime list) Syncing

### DISCUSSION THREAD

**Discussion by ov-ah** [pystardust/ani-cli#1294] [Ideas]
I have seen chrome extensions in the past that automatically sync your watch data (episode rating etc.) from your aniwatch or crunchyroll etc. to myanimelist.net which is very convenient so you don't have to manually update your progress 1 episode at a time. I think this would be a cool and relatively lightweight feature that could be added.

**Comment by ov-ah** [comment]
https://chromewebstore.google.com/detail/mal-sync/kekjfbackdeiabghhcdklcdoekaanoel
https://myanimelist.net

**Comment by port19x** [comment]
Jerry has anilist syncing. mal is old and crusty, why use that side

https://github.com/justchokingaround/jerry?tab=readme-ov-file#features

**Reply by ov-ah** [reply]
Not sure about others but everyone I know personally uses MAL, even though it might seem clunky its what most people have been using for a long time so why make the effort to transfer over to a new platform?

**Comment by ZanDev32** [comment]
[anipy-cli ](https://github.com/sdaqo/anipy-cli)has MAL syncing. Try it by yourself
